Tests/Twig/Loader/FilesystemLoaderTest.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Tests\Twig\Loader;
+
+use Liip\ThemeBundle\ActiveTheme;
+use Liip\ThemeBundle\Twig\Loader\FilesystemLoader;
+use Symfony\Bundle\FrameworkBundle\Templating\TemplateReference;
+
+class FilesystemLoaderTest extends \PHPUnit_Framework_TestCase
+{
+    public function testGetSourceContextWithLocator()
+    {
+        $activeTheme = new ActiveTheme('test', ['test']);
```

Comment:
feel free to drop PHP 5.3 and 5.4 from the test matrix .. or adjust to use the old array syntax